GVRf/Framework/src/org/gearvrf/GVRHybridObject.java

```diff
@@ -50,23 +51,41 @@
      */
     private long mNativePointer;
 
+    /**
+     * If true, the native object will be deleted by this class; otherwise,
+     * it won't be deleted.
+     */
+    private boolean mOwnNative;
```

Comment:
On first glance this is innocent enough but it is a symptom of the fact that the import process has not been abstracted enough. Assimp is now driving our design and we have to special-case the bone data by breaking a framework-wide assumption - that the Java objects own and manage the native peers, unconditionally.

This design is not set in stone but if it were to change, we need to see the general case and supply appropriate solution, not circumventing the current.